### ISSUE TITLE
Adding a gesture can make NVDA speak slower and pause between words, making it easier to understand

### ISSUE THREAD

**Issue by zh-yx** [nvaccess/nvda#15581]
Hi all!

I am a NVDA user from China. In China, there are many NVDA users who have poor English but need to listen to English content by screen reader. I am one of them.

They often face the problem of not being able to **hear** some **words** in English sentences clearly. Providing the following **ways** is expected to help them:

- Speak more slowly;

- Add pauses between words.

The above will hopefully help them hear every word clearly.

### Is your feature request related to a problem? Please describe.

At present, NVDA does not have gestures to read the current content at a slower pace.

There is also no gesture to add pauses between words to repeatedly report the current content.

### Describe the solution you'd like

Firstly, we can add a gesture **script**. This **script** is used to read the current content at a **slower** speed and add **pauses** between words.

In the **script**, we first obtain the current content **text**.

Next, we generate a speech **sequence** from the obtained **text**. Specifically, we will do this:

1. Add the `RateCommand` command at the beginning of the **sequence** to **slow down** the speech speed.

2. We split the text by words and add `BreakCommand` commands between the words.

Finally, we push the speech **sequence** into the synthesizer.

The code may look like this,

```python
	sequence = []
	sequence.append(speech.commands.RateCommand(offset=-30))
	for item in text.split():
		sequence.append(item)
		sequence.append(speech.commands.BreakCommand(200))
	speech.Speak(sequence)
```

I understand that this code is very rudimentary and may have various issues. But we can discuss based on this code.

### Describe alternatives you've considered

This may not be a problem.

We can improve it by writing a Add-on.

### Additional context

None

******

The above are some of my thoughts. I would be happy to hear everyone's discussion on this.

Thank you!

**Comment by seanbudd** [comment]
This exists in some sense: open the synth settings ring and adjust speech rate.
Pausing between speech would be misleading given how punctuation is handled.
This might be better suited to an add-on.
Closing as won't fix.
